examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2614,10 +2614,7 @@ class GLTFParser {
 			texture.wrapS = WEBGL_WRAPPINGS[ sampler.wrapS ] || RepeatWrapping;
 			texture.wrapT = WEBGL_WRAPPINGS[ sampler.wrapT ] || RepeatWrapping;
 
-			parser.associations.set( texture, {
-				type: 'textures',
-				index: textureIndex
-			} );
+			parser.associations.set( texture, { textures: textureIndex } );
```

Comment:
nit: I might change `textures` to `texture` and make the rest singular too, since we're only giving a single index.

Reply:
I guess the reason why they are plural is to easily access json data because they are plural in glTF json.

```
// The existing associations
const def = json[association.type][association.index];

// The new associations
for (const key in associations) {
  const def = json[key][associations[key]];
}
```